From: Michal Kubecek <mkubecek@suse.cz>
Date: Fri, 10 Jun 2016 10:00:06 +0200
Subject: kabi: prevent spurious modversion changes after bsc#982544 fix
Patch-mainline: Never, kabi workaround
References: bsc#982544

Including net/ip6_fib.h in net/bridge/br_private.h confuses the kabi
checker as some structures become defined. Hide the include from
genksyms the usual way.

Signed-off-by: Michal Kubecek <mkubecek@suse.cz>
---
 net/bridge/br_private.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/net/bridge/br_private.h b/net/bridge/br_private.h
index 8f8474fff03f..5ac51aadcd04 100644
--- a/net/bridge/br_private.h
+++ b/net/bridge/br_private.h
@@ -18,7 +18,9 @@
 #include <linux/netpoll.h>
 #include <linux/u64_stats_sync.h>
 #include <net/route.h>
+#ifndef __GENKSYMS__
 #include <net/ip6_fib.h>
+#endif
 #include <linux/if_vlan.h>
 
 #define BR_HASH_BITS 8
-- 
2.9.0

